src/common.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Copyright>Copyright © 2013-2021 Akka.NET Team</Copyright>
     <Authors>Akka.NET Team</Authors>
-    <VersionPrefix>1.4.28</VersionPrefix>
+    <VersionPrefix>1.4.32</VersionPrefix>
```

Comment:
I believe this was automatically incremented but not by me!

Reply:
This is done by the build system